DataConnectors/ZeroNetworks/AccessOrchestratorFunctionConnector/DataConnector_API_AzureFunctionApp_ZeroNetworks_AccessOrchestrator_Audit.json

```diff
@@ -0,0 +1,141 @@
+			},
+			{
+				"name": "REST API Credentials",
+				"description": "**Zero Networks Access Orchestrator** **API Token** is required for REST API. See the API Guide."
+			}
+        ]
+    },
+    "instructionSteps": [
+        {
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Zero Networks REST API to pull its logs into Microsoft Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - Configuration steps for the Zero Networks API**\n\n See the API Guide to obtain the credentials. \n"
+		},
+		{
+			"title": "",
+			"description": "**STEP 2 - Choose ONE from the following two deployment options to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Workspace data connector, have the Workspace ID and Workspace Primary Key (can be copied from the following).",
+			"instructions": [{
+					"parameters": {
+						"fillWith": [
+							"WorkspaceId"
+						],
+						"label": "Workspace ID"
+					},
+					"type": "CopyableLabel"
+				},
+				{
+					"parameters": {
+						"fillWith": [
+							"PrimaryKey"
+						],
+						"label": "Primary Key"
+					},
+					"type": "CopyableLabel"
+				}
+			]
+		},
+		{
+			"title": "Option 1 - Azure Resource Manager (ARM) Template",
+			"description": "Use this method for automated deployment of the Zero Networks Access Orchestrator Audit data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FDataConnectors%2FZeroNetworks%2FAccessOrchestratorFunctionConnector%2Fazuredeploy_Connector_ZeroNetworks_AccessOrchestrator_Audit_AzureFunction.json)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n> **NOTE:** Within the same resource group, you can't mix Windows and Linux apps in the same region. Select existing resource group without Windows apps in it or create new resource group.\n3. Enter the **APIToken** and deploy. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
@dicolanl  Please update the azuredeploy short link https://aka.ms/sentinel-ZeroNetworks-azuredeploy

Reply:
Done